.env

```diff
@@ -0,0 +1,2 @@
+GOOGLE_KEY=AIzaSyDB7tZtZq6ukHjJ1TdMYgQyjyOPINC0WVU
```

Comment:
.gitignore didn't ignore this file?

Reply:
I went ahead and deleted these, thanks for the catch.